static/js/modules/nootils.js

```diff
@@ -19,8 +19,8 @@ module.exports = function(settings) {
       } else {
```

Comment:
My understandings:

 - `.getTime()` returns the number of millis since 1970 UTC. So it doesn't care about timezones.

 - `new Date()` creates a date by storing the number of millis since 1970 UTC, but formats and is interacted with based on the system timezone.

So `result` ends up being the local representation of whatever date was passed (which will *also* probably be the local representation depending on how it was created from the database, but this code being defensive and re-creating it with the # millis since 1970 UTC is not bad).